salt/metalk8s/addons/dex/deployed/dex-conf.sls

```diff
@@ -0,0 +1,75 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+
+{%- set oidc_service_ip = salt.metalk8s_network.get_oidc_service_ip() %}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app.kubernetes.io/instance: dex
+    app.kubernetes.io/managed-by: salt
+    app.kubernetes.io/name: dex
+    app.kubernetes.io/part-of: metalk8s
+    app.kubernetes.io/version: "2.19.0"
+    helm.sh/chart: dex-2.4.0
+    heritage: metalk8s
+  name: dex
+  namespace: metalk8s-auth
+stringData:
+  config.yaml: |-
+      issuer: 'https://{{ oidc_service_ip }}:32000'
```

Comment:
What would it imply to mention use the Ingress in this config file? Does it only affect what is exposed through `.well-know/openid-configuration`?

```suggestion
      issuer: 'https://{{ grains.metalk8s.control_plane_ip }}:8443/oidc'
```